doc/man/8/rbd.rst

```diff
@@ -532,14 +532,15 @@ Commands
   disabled on all images (within the pool or namespace) for which mirroring
   was enabled, whether by default or explicitly.
 
-:command:`mirror pool enable` [*pool-name*] *mode*
+:command:`mirror pool enable` [*pool-name*] *mode* [--remote-namespace]
   Enable RBD mirroring within a pool or namespace.
   The mirroring mode can either be ``pool`` or ``image``.
   If configured in ``pool`` mode, all images in the pool or namespace
   with the journaling feature enabled are mirrored.
   If configured in ``image`` mode, mirroring needs to be
   explicitly enabled (by ``mirror image enable`` command)
-  on each image.
+  on each image. A namespace can be mirrored to a different namespace on
+  the remote cluster using the ``--remote-namespace`` option.
```

Comment:
Nit: Start the sentence on a new line for consistency with the rest of this command's description:
```suggestion
  on each image.
  A namespace can be mirrored to a different namespace on
  the remote cluster using the ``--remote-namespace`` option.
```